marl_eval/json_tools/json_utils.py

```diff
@@ -135,28 +136,42 @@ def pull_neptune_data(
     if not os.path.exists(store_directory):
         os.makedirs(store_directory)
 
+    # Suppress neptune logger
+    neptune_logger = logging.getLogger('neptune')
+    neptune_logger.setLevel(logging.ERROR)
+
+    # Initialise a counter to ensure unique file names
+    counter = 0
+
     # Download and unzip the data
     for run_id in tqdm(run_ids, desc="Downloading Neptune Data"):
         run = neptune.init_run(project=project_name, with_id=run_id, mode="read-only")
         for data_key in run.get_structure()[neptune_data_key].keys():
+            counter += 1
             file_path = f"{store_directory}/{data_key}"
```

Comment:
If we name the files with the counter then we do not have to rename later on. 
```suggestion
            file_path = f"{store_directory}/{data_key}_{i}"
```